### PR TITLE
pfetch-rs: init at 2.7.0

### PR DESCRIPTION
**Pull request by Gobidev**
This PR adds a new package for pfetch-rs, a rewrite of the pfetch system information tool in Rust.

It also adds myself as a maintainer of this package.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes (or backporting 23.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/pfetch-rs/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "pfetch";
```
Comment (Lord-Valen) [inline pkgs/tools/misc/pfetch-rs/default.nix:4]:
```suggestion
  pname = "pfetch-rs";
```

File: pkgs/tools/misc/pfetch-rs/default.nix [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  version = "2.7.0";
+
+  src = fetchFromGitHub {
+    owner = "Gobidev";
+    repo = "pfetch-rs";
```
Comment (Lord-Valen) [inline pkgs/tools/misc/pfetch-rs/default.nix:9]:
```suggestion
    repo = pname;
```

**Review by Lord-Valen — CHANGES_REQUESTED** [review]
Please split the change to maintainers/maintainer-list.nix into its own separate commit called `maintainers: add gobidev`.

Two commits total:

`maintainers: add gobidev`
`pfetch-rs: init at 2.7.0`

**Comment by Gobidev** [comment]
I added the requested changes

**Comment by Lord-Valen** [comment]
Result of `nixpkgs-review pr 237819` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>pfetch-rs</li>
  </ul>
</details>

**Comment by Lord-Valen** [comment]
Works as expected on x86_64-linux.

**Review by Lord-Valen — APPROVED** [review]

File: pkgs/tools/misc/pfetch-rs/default.nix [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+rustPlatform.buildRustPackage rec {
+  pname = "pfetch-rs";
+  version = "2.7.0";
+
+  src = fetchFromGitHub {
+    owner = "Gobidev";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0b5300g7q9f07pcr5vy42v6wgb8sk6glai3kb5pj7f9ilq87mmr8";
```
Comment (drupol) [inline pkgs/tools/misc/pfetch-rs/default.nix:11]: Use `hash` in place of `sha256` please.

**Review by drupol — CHANGES_REQUESTED** [review]

**Review by drupol — APPROVED** [review]

**Comment by drupol** [comment]
@GrahamcOfBorg build pfetch-rs

**Review by Lord-Valen — APPROVED** [review]

**Pull request merged**